lib/mutant/matcher/config.rb

```diff
@@ -22,6 +38,45 @@ def add(attribute, value)
         update(attribute => public_send(attribute).dup << value)
       end
 
+    private
+
+      # Present attributes
+      #
+      # @return [Array<Symbol>]
+      #
+      # @api private
+      def present_attributes
+        to_h.reject { |_key, value| value.empty? }.map(&:first)
```

Comment:
Are you sure? http://ruby-doc.org/core-2.2.0/Hash.html#method-i-reject

Even in 1.9.3 it returns a Hash: http://ruby-doc.org/core-1.9.3/Hash.html#method-i-reject